cli/metadata/metadata.go

```diff
@@ -0,0 +1,21 @@
+package metadata
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+func AppendBuildMetadata(args, originalArgs []string) ([]string, error) {
+	// Ensure arg0 is "bb" (by default, it is an abs path)
+	originalArgs = append([]string{"bb"}, originalArgs[1:]...)
```

Comment:
Could hard-coding "bb" make it harder for folks that are using the .bazelversion approach?

Reply:
That's a good point, I'll try to see if there's a way to show "bazel" in that case.

Reply:
Done (showing "bazel" now if invoked via .bazelversion).